GVFS/GVFS.Installer.Mac/scripts/Distribution_GVFS.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<installer-gui-script minSpecVersion="1">
+    <pkg-ref id="com.vfsforgit.pkg"/>
+    <options customize="never" allow-external-scripts="true"/>
+    <choices-outline>
+        <line choice="default">
+            <line choice="com.vfsforgit.pkg"/>
+        </line>
+    </choices-outline>
+    <choice id="default"/>
+    <choice id="com.vfsforgit.pkg" visible="false">
+        <pkg-ref id="com.vfsforgit.pkg"/>
+    </choice>
+    <pkg-ref id="com.vfsforgit.pkg" version="1.2.3" onConclusion="none">VFSFORGITPACKAGEFILENAME</pkg-ref>
+    <installation-check script="InstallationCheck()"/>
+    <script>
+        function InstallationCheck(prefix) 
+        {
+            var blockingProcesses = GetBlockingProcesses();
+            if (blockingProcesses.length > 0)
+            {
+                my.result.message = "Quit these processes and try again - " + blockingProcesses.join(", ") + ".";
+                my.result.type = 'Fatal';
+                return false;
+            }
+            
+            my.result.message = "No blocking processes running.";
+            my.result.type = "Info";
+            return true;
+        }
+        
+        function GetBlockingProcesses()
+        {
+            var watchList = [ "Calculator", "TextEdit", "Preview" ];
+            var blockingProcesses = new Array();
+            for (var process of watchList)
+            {
+                var isRunningCmd = "[ `ps -ef | grep -w '" + process + "' | grep -v grep | wc -l` -eq 0 ]";
+                // var isRunningCmd = "exit 1;";
+                var status = system.run("/bin/bash", "-c", isRunningCmd);
+                system.log("isRunningCmd: " + isRunningCmd + ". Status: " + status);
```

Comment:
Where is `system.log` written?

Reply:
That shows up in the Installer logs. Like this - 
 
<img width="494" alt="screen shot 2019-03-06 at 11 38 23 am" src="https://user-images.githubusercontent.com/378580/53897603-7ba57a80-4004-11e9-9099-8714dcd994c6.png">

Reply:
Thanks, is that log also saved to disk somewhere?  Or is it only present while the installer is running?

Reply:
User can choose `File->Save` from the installer menu and save it to disk.